### PR TITLE
update 15 CVEs

### PR DESCRIPTION
**Pull request by CERT-VDE**

### CODE REVIEW

File: 2021/34xxx/CVE-2021-34604.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
```
Comment (cve-team) [inline 2021/34xxx/CVE-2021-34604.json]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2021/34xxx/CVE-2021-34604.json]: ('Invalid control character at: line 6 column 27 (char 122)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/34xxx/CVE-2021-34604.json (hunk 1 above)
Comment (CERT-VDE) [reply 2021/34xxx/CVE-2021-34604.json]: fixed in 2nd commit

Comment (CERT-VDE) [reply 2021/34xxx/CVE-2021-34604.json]: fixed in 2nd commit

**Pull request merged**